### PR TITLE
Update: change weekly pipeline releaseType from "Release" to "Weekly"

### PR DESCRIPTION
**Pull request by zdtsw**
Ref: https://github.com/adoptium/ci-jenkins-pipelines/issues/637

After this PR:
change on standard weekly (not evaluation-weekly) will be from release=true to release=false, publish is still the same "true"
So the behavior is set to the same as the evaluation-weekly
weekly will trigger remote TCK , same as when releaseType == Release

I keep the logic of upload artifacts which was planned as input for comparison job
but if we do not set releaseType as release for weekly, then the build result wont be able to do comparison. 

This would require https://github.com/adoptium/jenkins-helper/pull/51# to get merged at the same time.

### CODE REVIEW

**Review by karianna — APPROVED** [review]

**Comment by sophia-guo** [comment]
Remote triggers are triggered by both weekly and release build with `releaseType=Release`. So the remote trigger logic will also need to be updated.

**Comment by zdtsw** [comment]
> Remote triggers are triggered by both weekly and release build with `releaseType=Release`. So the remote trigger logic will also need to be updated.

@andrew-m-leonard is the plan to keep having weekly trigger "remote TCK" as well? 
if yes, then we need to update https://github.com/adoptium/ci-jenkins-pipelines/blob/94ad798cef7807886673a0310cf12f24b3867d0f/pipelines/build/common/openjdk_build_pipeline.groovy#L1864 to include "Weekly".

**Comment by smlambert** [comment]
re: https://github.com/adoptium/ci-jenkins-pipelines/pull/639#issuecomment-1456242935, we want to run TCKs weekly.

**Comment by zdtsw** [comment]
> re: [#639 (comment)](https://github.com/adoptium/ci-jenkins-pipelines/pull/639#issuecomment-1456242935), we want to run TCKs weekly.

Will fix that piece. mark as Draft for now

File: pipelines/src/test/groovy/IndividualBuildConfigTest.groovy [hunk 1]
```diff
@@ -40,6 +43,7 @@ class IndividualBuildConfigTest {
                 ADDITIONAL_FILE_NAME_TAG   : 'k',
                 JDK_BOOT_VERSION           : 'l',
                 RELEASE                    : false,
+                WEEKLY                     : false,
```
Comment (andrew-m-leonard) [inline pipelines/src/test/groovy/IndividualBuildConfigTest.groovy:46]: We need to add WEEKLY to the jenkins-helper definition: https://github.com/adoptium/ci-jenkins-pipelines/pull/727/files

Comment (andrew-m-leonard) [reply pipelines/src/test/groovy/IndividualBuildConfigTest.groovy:46]: I really should read all the comments! It's here: https://github.com/adoptium/jenkins-helper/pull/51/files
Merged now

**Comment by andrew-m-leonard** [comment]
run tests

**Comment by andrew-m-leonard** [comment]
run tests

**Comment by zdtsw** [comment]
looks strange, two failed job on win64 indicated something like
"17:17:05  checkoutRequiredCodeToBuild succeeded
17:17:09  warning: failed to remove build/windows-x86_64-server-release/hotspot/variant-server/gensrc/adfiles/ad_x86.hpp: Device or resource busy
17:17:09  warning: failed to remove build/windows-x86_64-server-release/hotspot/variant-server/libjvm/objs/BUILD_LIBJVM.pch: Device or resource busy"

**Comment by zdtsw** [comment]
run test quick 8,17

**Comment by andrew-m-leonard** [comment]
> looks strange, two failed job on win64 indicated something like "17:17:05 checkoutRequiredCodeToBuild succeeded 17:17:09 warning: failed to remove build/windows-x86_64-server-release/hotspot/variant-server/gensrc/adfiles/ad_x86.hpp: Device or resource busy 17:17:09 warning: failed to remove build/windows-x86_64-server-release/hotspot/variant-server/libjvm/objs/BUILD_LIBJVM.pch: Device or resource busy"

That's usually a Windows "locked file" issue..

**Comment by andrew-m-leonard** [comment]
run test quick 8,17

**Comment by andrew-m-leonard** [comment]
run test quick 8,17

**Review by andrew-m-leonard — APPROVED** [review]
I think looks good now

**Comment by andrew-m-leonard** [comment]
run tests quick 8,17

**Pull request merged**